### ISSUE TITLE
Flathub Verification

### ISSUE THREAD

**Issue by Roman19S** [crow-translate/crow-translate#674]
**Describe the problem or limitation you are having**
The flatpak is unverified so people who find it through Flathub wont know it's official

**Describe the solution you'd like**
You can verify the app through the website or repo

**Describe alternatives you've considered**
You can also contact Flathub to get manually verified

**Additional context**
Here's a link to Flathub's verification documentation for Developers https://docs.flathub.org/docs/for-app-authors/verification/

**Comment by Shatur** [comment]
@gasinvein is helping me with deploying the app. Do I need to invite you as a collaborator in order to pass verification?

**Comment by Roman19S** [comment]
No, you just need someone with access to the Flathub login and either the the repo owner or someone with website access. Just [log in to Flathub](https://www.flathub.org/login). Go to the developer tab, if you're not already there. Click the "Developer Settings" button under the app. At the top of the page, find the "Setup Verification" section. The instructions there will walk you through the verification process.

**Comment by Shatur** [comment]
@Roman19S but I didn't publish Crow on FlatHub, @gasinvein did.

**Comment by Shatur** [comment]
Ah, I get it! Thank you. Not quite familiar with FlatHub.
Looks like we point to a wrong website, instead of `crow-translate.io` it should be `crow-translate.github.io`. How can I change it?

**Comment by Roman19S** [comment]
![image](https://github.com/crow-translate/crow-translate/assets/30319719/6b62b812-522b-42a3-82b1-d03358f94ae8)
It's showing up correctly on Flathub. Where is the website incorrect?

**Comment by Shatur** [comment]
Interesting, but on verification it asks me about the following:
![image](https://github.com/crow-translate/crow-translate/assets/22453358/a86b108e-b694-4fc2-a2e4-d6f4d6307a52)

**Comment by Roman19S** [comment]
I think I see the issue. It's your app's id 
![image](https://github.com/crow-translate/crow-translate/assets/30319719/90e1c499-483c-4d8d-bdec-5b29a3568db4)

**Comment by Shatur** [comment]
Ah, so it should be `io.github.crow_translate.CrowTranslate`?
How can I change it?

**Comment by Roman19S** [comment]
https://discourse.flathub.org/t/how-to-change-an-app-id/124
